ImageFeed/Helpers/UIColor+Extensions.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  UIColor+Extensions.swift
+//  ImageFeed
+//
+//  Created by Valeriy on 09.05.2023.
+//
+
+import UIKit
+
+extension UIColor {
+    static var ypBlack: UIColor { UIColor(named: "YP Black") ?? UIColor.black }
+    static var ypGray: UIColor { UIColor(named: "YP Gray") ?? UIColor.gray }
+    static var ypBackground: UIColor { UIColor(named: "YP Background") ?? UIColor.darkGray }
+    static var ypWhite: UIColor { UIColor(named: "YP White") ?? UIColor.white }
+    static var ypWhiteAlpha50: UIColor { UIColor(named: "YP White (Alpha 50)") ?? UIColor.white.withAlphaComponent(0.5)}
```

Comment:
практически всегда у фигурной скобки оставляются пробельные отступы 
```suggestion
    static var ypWhiteAlpha50: UIColor { UIColor(named: "YP White (Alpha 50)") ?? UIColor.white.withAlphaComponent(0.5) }
```